### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ChenzIsHomo**

### CODE REVIEW

**Comment by ChenzIsHomo** [comment]
also musically allows logging in

File: README.md [hunk 1]
```diff
@@ -163,6 +164,7 @@ wsa://com.android.settings
 | Monument Valley | 2.7.17 | ✔️ |||
 | Monument Valley 2 | 2.0.3 | ✔️ |||
 | MT File Manager | 2.10.0 | ✔️ |||
+| Musically (TikTok) | 7.8.0 | ✔️ |||
```
Comment (pratyakshm) [inline README.md:167]: Can you please clarify the difference between Musically (TikTok) and these?
image

Comment (ChenzIsHomo) [reply README.md:167]: musically is a really old version of tiktok, before it was tiktok.

Comment (ChenzIsHomo) [reply README.md:167]: also musically allows signing in with a tiktok account but regular tiktok doesnt

Comment (pratyakshm) [reply README.md:167]: Alright.

**Comment by pratyakshm** [comment]
Thanks!

**Pull request merged**